logging/loggingtest/logger.go

```diff
@@ -123,10 +123,7 @@ func New() *Logger {
 				lw.count(req)
 			case c := <-clear:
 				lw.clear()
-				select {
-				case c <- struct{}{}:
-				default:
```

Comment:
https://github.com/zalando/skipper/blob/233e1d33f24e0f3ef062488fbb99bfb187c1cebf/logging/loggingtest/logger.go#L187-L192

Failure sequence:
```
 // Reset(): sends return channel
tl.clear <- ch

// New(): can not send on return channel because 
// the channel has no buffer and there is no receiver, 
// the return channel is lost
select default 

 // Reset(): hangs
<-ch
```
Example https://github.com/zalando/skipper/pull/1552/checks?check_run_id=1261043711